committee/wg-governance/wg-governance-zh_CN.md

```diff
@@ -0,0 +1,40 @@
+# TiKV Working Group 治理规范
+
+## Working Group 是什么？
+
+Working Group 提供一个群组，来让社区的贡献者可以针对某个问题进行一段时间的合作。因为 Working Group 通常是解决一个跨越多个 Special Interest Group（SIG）职责范围的问题，所以它通常会由来自不同 SIG 的成员或者一些其他社区贡献者组成。Working Group 通常会在问题被解决之后解散，在 Working Group 中创建的成果，或者成员，可以后续被 SIG 接收。更详细的创建或者解散程序，可以参见 [WG Lifecycle](#wg-lifecycle)
+
+Working Group 具有以下特征：
+
+* 不拥有对代码的所有权
+* 有明确的可以衡量的目标，以及可以被交付的结果
+* 是一个短期的组织，并且会在目标或者使命被完成以后被解散
+
+## Working Group 与 Special Interest Group 的关系
```

Comment:
Shall we add [this picture](https://download.pingcap.com/images/blog-cn/tidb-community-upgrade/5.png) here for a better idea of the relationship between Working Group and Special Interest Group?